cli/azd/cmd/init.go

```diff
@@ -187,9 +187,9 @@ func (i *initAction) Run(ctx context.Context, cmd *cobra.Command, args []string,
 		}
 
 		if len(duplicateFiles) > 0 {
-			fmt.Printf("warning: the following files will be overwritten with the versions from the template: \n")
+			fmt.Fprintf(cmd.OutOrStdout(), "warning: the following files will be overwritten with the versions from the template: \n")
 			for _, file := range duplicateFiles {
-				fmt.Printf(" * %s\n", file)
+				fmt.Fprintf(cmd.OutOrStdout(), " * %s\n", file)
```

Comment:
Maybe this could be handled from the `console.Message` ,

Reply:
I actually started that at one point but realized that `console.Message` didn't always write messages (it had some logic to decide if it should or not based on output types) whereas this always would have written it, so I stuck with that for now.  I agree this is something we maybe want to allow, and it feeds into that deeper conversation we are having about io.Writer vs input.Console